<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGLCM</class>
 <widget class="QDialog" name="DialogGLCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>GLCM Setting</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>320</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>100</y>
     <width>91</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Theta</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>41</y>
      <width>60</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButtonD0">
       <property name="text">
        <string>&amp;0</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonD45">
       <property name="text">
        <string>&amp;45</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonD90">
       <property name="text">
        <string>&amp;90</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonD135">
       <property name="text">
        <string>&amp;135</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>30</y>
     <width>91</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Gray Level</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>41</y>
      <width>60</width>
      <height>230</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonL8">
       <property name="text">
        <string>&amp;8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonL16">
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonL32">
       <property name="text">
        <string>&amp;32</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonL64">
       <property name="text">
        <string>&amp;64</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonL128">
       <property name="text">
        <string>128</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonL256">
       <property name="text">
        <string>&amp;256</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>51</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelDistance">
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxDistance">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGLCM</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGLCM</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
